Sprint-3/1-key-implement/3-get-card-value.js

```diff
@@ -33,7 +33,11 @@ assertEquals(aceofSpades, 11);
 // When the function is called with such a card,
 // Then it should return the numeric value corresponding to the rank (e.g., "5" should return 5).
 const fiveofHearts = getCardValue("5♥");
-// ====> write your test here, and then add a line to pass the test in the function above
+// ====> function getCardValue(card) {
+ // const rank = card[0]; // Extracts the first character (rank) from the card string
+ // if (!isNaN(rank)) return Number(rank); 
+//}
+
 
```

Comment:
getCardValue function is not working as expected. Please integrate desired logic inside of the function